.github/workflows/gh-pr.yml

```diff
@@ -4,14 +4,26 @@ on:
   pull_request:
     branches:
       - master
+      - v6
```

Comment:
btw we might be more generic here and use a pattern, for instance,
`- 'releases/**'`
as that v6 would be `releases/v6` with auto publishing